src/js/views/patients/schedule/schedule-list.scss

```diff
@@ -115,8 +122,8 @@
   width: 48px;
 }
 
-.schedule-list__state-patient,
-.schedule-list__action-name {
+.schedule-list__patient-details,
+.schedule-list__action-list-cell {
```

Comment:
This makes overflowing text for the `State, Action` table cell be hidden with an ellipsis (`...`).

<img width="1023" alt="Screen Shot 2022-08-08 at 11 03 56 AM" src="https://user-images.githubusercontent.com/35355575/183474762-1c033135-eb6c-4757-8753-5b25ace702e7.png">

Before this was only applied to the action name (`.schedule-list__action-name`). But since the state icon is now in that table column, it needs to be applied to the entire table column instead (`.schedule-list__action-list-cell`).